nixpkgs_review/cli/pr.py

```diff
@@ -78,8 +78,14 @@ def pr_command(args: argparse.Namespace) -> str:
                 warn(f"https://github.com/NixOS/nixpkgs/pull/{pr} failed to build: {e}")
         assert review is not None
 
+        all_succeeded = True
         for pr, path, attrs in contexts:
-            review.start_review(attrs, path, pr, args.post_result, args.print_result)
+            all_succeeded &= review.start_review(
+              attrs, path, pr, args.post_result, args.print_result
+            )
```

Comment:
If you prefer less procedural logic and names, then this is the more pythonic way, which does not make use of bitwise boolean operators:

```python
all_succeeded = all(
    review.start_review(attrs, path, pr, args.post_result, args.print_result)
    for pr, path, attrs in contexts
)
```